### DISCUSSION TITLE
How to remove the "BEL"/ "END" words on console

### DISCUSSION THREAD

**Discussion by NxDude** [kingToolbox/WindTerm#522] [Q&A] [answered]
In serial port console,  when I click the tab button, it would show "BEL" words on console.
how to remove it

thanks

**Comment by kingToolbox** [comment]
Hello, can you collect the debug log for me for further analysis, if possible, the steps are as follows:

- Open your serial session.
- Check the menu item` Menubar - Mode (7) - Debug Mode`.
- **Click the tab button twice, and make sure that the `BEL` words are shown every time**.
- Uncheck the menu item `Menubar - Mode (7) - Debug Mode`.
- Click the menu item `Right click menu - Log - Open Log Folder`.
- Open the folder `debug` and paste your log file in your comment here. You can zip it before upload.

The log content is in plain text and will not contain any private information. You can use any editor to view it. Thank you.

**Comment by NxDude** [comment]
Hello, please help to check the debug log 
I can see some HEX format and some stange words "[5D", "[J" when I try to using tab to auto-completion.
[debug.zip](https://github.com/kingToolbox/WindTerm/files/8085979/debug.zip)

BEL -> 0x07
END ->0x1b
BS -> 0x08

**Comment by NxDude** [comment, marked as answer]
Hello,

I already fix this issue.

the serial setting should choose Data -> Text, not using defualt Binary.
then reconnect the serial session. 

It works.

Thanks

**Comment by kingToolbox** [comment]
Very good news. Welcome to continue submitting new feature requests and bugs, thank you.
